src/components/App.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react'
```

Comment:
We need to decide if we should add spaces between curly braces and their contents in non-React code. Personally, I like it more when the spaces are there (`{ Component }`).
Also, since we've already imported React, I think we shouldn't import Component on top of it. Instead, do `export class App extends React.Component`.